packages/vm/lib/evm/opcodes/functions.ts

```diff
@@ -469,15 +469,18 @@ export const handlers: Map<number, OpHandler> = new Map([
       const [memOffset, dataOffset, dataLength] = runState.stack.popN(3)
 
       subMemUsage(runState, memOffset, dataLength)
```

Comment:
Can you unify the names for the length along these operations as some minor refactoring along the changes here (here it is `dataLength`, on other places `length`)?

Reply:
Will also clean this